WcaOnRails/config/locales/en.yml

```diff
@@ -543,7 +542,11 @@ en:
         reason:
           assigned: "you have a WCA ID assigned"
           registered: "you have registered for a competition"
+      guidelines_confirmation: "I confirm that my avatar fulfils all avatar guidelines."
```

Comment:
nit: fulfil -> fulfill

I didn't realize this was another US English vs the rest of the world thing: http://grammarist.com/spelling/fulfil-fulfill/!